### PR TITLE
feat: change CPU benchmark to use multiple columns

### PR DESCRIPTION
**Pull request by coryan**
To be fair, also separate the response via stubs into "rows", and fix some problems
with session setup on the Stub case.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp-spanner/1076)

### CODE REVIEW

File: google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc [hunk 1]
```diff
@@ -139,6 +139,9 @@ int main(int argc, char* argv[]) {
       google::cloud::spanner_testing::RandomDatabaseName(generator));
   config.database_id = database.database_id();
 
+  // Once the configuration is and the database name set, print everything out.
```
Comment (devbww) [inline google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc]: "Once the configuration is" what?

File: google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc [hunk 2]
```diff
@@ -402,20 +418,37 @@ class ReadExperiment : public Experiment {
...
-        if (!result.chunked_value()) {
-          row_count += result.values_size() / 2;
+        if (result.chunked_value()) {
+          // We do not handle chunked values in the benchmark.
+          continue;
+        }
+        row.resize(columns.size());
+        std::size_t index = 0;
+        for (auto& value : *result.mutable_values()) {
+          row[index++] = std::move(value);
```
Comment (devbww) [inline google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc]: Opt: move post ++ to pre ++ on next line?

Comment (coryan) [reply google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc]: Good idea, fixed.

File: google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc (hunk 1 above)
Comment (coryan) [reply google/cloud/spanner/benchmarks/multiple_rows_cpu_benchmark.cc]: Thanks, fixed.

**Pull request merged**